.travis.yml

```diff
@@ -67,7 +67,7 @@ matrix:
         on:
           tags: true
     - name: "Push image to Docker Hub"
-      python: 3.5
+      python: 3.6
```

Comment:
Can you please add testing `3.8-dev`

Reply:
looks like numpy is not available for python 3.8.
https://travis-ci.com/src-d/ml-core/jobs/221278241

Reply:
Nope. Please look at the error.

```
numpy.distutils.system_info.NotFoundError: No lapack/blas resources found. 
```

You are building (not missing) scipy (not numpy) and there is no BLAS installed.

The reason is because scipy [does not have 3.8 wheels yet](https://pypi.org/project/scipy/#files) and has to be built.

scipy takes ~10 minutes to be built. So we cannot add 3.8 now.